packages/expo-image/android/src/main/java/expo/modules/image/ExpoImageViewWrapper.kt

```diff
@@ -170,13 +171,32 @@ class ExpoImageViewWrapper(context: Context, appContext: AppContext) : ExpoView(
 
   fun setIsAnimating(setAnimating: Boolean) {
     val resource = activeView.drawable
+    /* Animatable animations always start from the beginning when resumed.
+    So we check first if the resource is a GifDrawable, because it can continue
+    from where it was paused. */
```

Comment:
```suggestion
    // Animatable animations always start from the beginning when resumed.
    // So we check first if the resource is a GifDrawable, because it can continue
    // from where it was paused.
```